specification/grpc/a2a.proto

```diff
@@ -581,43 +581,40 @@ message SendMessageRequest {
 }
 
 message GetTaskRequest {
-  // name=tasks/{id}
-  string name = 1 [(google.api.field_behavior) = REQUIRED];
+  string id = 1 [(google.api.field_behavior) = REQUIRED];
   int32 history_length = 2;
 }
 
 message CancelTaskRequest {
-  // name=tasks/{id}
-  string name = 1;
+  string id = 1;
 }
 
 message GetTaskPushNotificationConfigRequest {
-  // name=tasks/{id}/pushNotificationConfigs/{push_id}
-  string name = 1;
+  string task_id = 1 [(google.api.field_behavior) = REQUIRED];
+  string config_id = 2 [(google.api.field_behavior) = REQUIRED];
 }
 
 message DeleteTaskPushNotificationConfigRequest {
-  // name=tasks/{id}/pushNotificationConfigs/{push_id}
-  string name = 1;
+  string task_id = 1 [(google.api.field_behavior) = REQUIRED];
+  string config_id = 2 [(google.api.field_behavior) = REQUIRED];
 }
 
 message CreateTaskPushNotificationConfigRequest {
   // The task resource for this config.
-  // Format: tasks/{id}
-  string parent = 1 [(google.api.field_behavior) = REQUIRED];
+  string id = 1 [ (google.api.field_behavior) = REQUIRED];
+  // BUG?: config_id is not referenced in the google.api.http
+  // spec for CreateTaskPushNotificationConfig.
```

Comment:
Someone else should weigh in, but I'm pretty sure that this message should not have a config_id.